src/thunderbots/software/test/ai/navigator/util.cpp

```diff
@@ -0,0 +1,87 @@
+#include "ai/navigator/util.h"
+
+#include <gtest/gtest.h>
+
+#include <cmath>
+#include <math.h>
+#include <cstdlib>
+#include <ctime>
+#include <iostream>
+#include <sstream>
+
+#include "geom/point.h"
+
+// Set this to 1 to enable debug output.
+#define DEBUG 0
+
+#if DEBUG
+#define dbgout std::cout
+#else
+std::ostringstream dbgout;
+#endif
+
+TEST(NavUtilTest, seg_vel_parallel)
+{
+    Point testp1, testp2, testp3;
```

Comment:
Quick comment at the top of the test indicating exactly what we're testing here would be really helpful! Same goes for the other tests cases here.

Reply:
Or a more verbose test name, eg `test_segments_in_parallel`. Test names can be extremely verbose since then it's obvious what the test was trying to check (if it gets reported in a failure), and it's not a variable you have to type over and over anyway.